config/rbac/role.yaml

```diff
@@ -11,19 +11,14 @@ rules:
   resources:
   - serviceaccounts
   verbs:
-  - create
   - get
   - list
-  - patch
-  - update
   - watch
 - apiGroups:
   - admissionregistration.k8s.io
   resources:
   - mutatingwebhookconfigurations
   verbs:
-  - create
-  - delete
   - get
   - list
   - patch
```

Comment:
Do we need patch and update?

Reply:
Just `UPDATE` is required. I've updated the PR!